Javascript-2/Week-1/Movie App/script.js

```diff
@@ -0,0 +1,201 @@
+import { movies } from "./arrayList.js";
+
+//////////////////////Streaming Function
+const nowStreaming = document.querySelector(".streaming");
+
+function createMovieCard(movie) {
```

Comment:
This function is longer than 110 lines; it is way too long. You need to split it into smaller parts. This will simplify your work for debugging and allow your coworkers to easily understand your logic.